packages/modules/devices/solax/solax/bat.py

```diff
@@ -10,27 +10,27 @@
 from modules.common.modbus import ModbusDataType
 from modules.common.simcount import SimCounter
 from modules.common.store import get_bat_value_store
-from modules.devices.solax.solax.config import SolaxBatSetup
+from modules.devices.solax.solax.config import SolaxBatSetup, Solax
 
 
 class SolaxBat(AbstractBat):
     def __init__(self,
-                 device_id: int,
+                 device_config: Union[Dict, Solax],
```

Comment:
```suggestion
                 device_config: Solax,
```
Das Dictionary wird in der Subdata in eine Klasse konvertiert.